package.json

```diff
@@ -175,8 +175,8 @@
     "react-native-launch-arguments": "^4.0.2",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
-    "react-native-nitro-modules": "0.25.2",
```

Comment:
Can we update nitro in a separate PR to simplify/clarify test steps, and reduce the risk of regressions or reverts?